net-print/cnijfilter-drivers/cnijfilter-drivers-3.80.ebuild

```diff
@@ -0,0 +1,167 @@
+# Copyright 1999-2012 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+MY_PN="${PN/-drivers/}"
+
+inherit eutils autotools flag-o-matic multilib
+
+DESCRIPTION="Canon InkJet Printer Driver for Linux (Pixus/Pixma-Series)."
```

Comment:
This is not a proper sentence, so no `.` at the end.

Reply:
Fixed